### PR TITLE
Adds new Nanomesh Uniforms Part1

### PR DESCRIPTION
**Pull request by Xantholne**
Adds the new Nanomesh Uniforms or NU for short. Available to ~90% of all jobs. 

Nanomesh Uniforms or NUs are uniforms  made of a Synthetic Nanofibre woven into each uniform  and designed specifically for Nanotrasen Vessel crews.

These uniforms feature several features including a epaulette on the vast majority of these uniforms  signifying that persons rank among the station. They also have an assortment of badges to show their qualification; i.e. a Scientopia Purple on Science Department uniforms for instance on the far right. Head of Staff in these department have additional badges signifying their knowledge in said department as well as a Command badge in place of the normal department qualification badge slot. High level positions such as the Captain and Representative have a Golden badge as they answer directly to Central Command.

All uniforms also have a waist guard being strapped on from the front which ensures the uniforms are as form fitting as possible to each individual.

Outside of the detailing; these uniforms are primarily made as Nanotrasen counterparts to the SolGov uniforms people love so much to give more of a chain of command.

Part 2 update if these get merged will be including uniforms currently not added for some jobs and probably some reshading as well as the suit slot item to work with labcoats and the likes.

776899cff0ad580e3e3a19349370b98e
15b614ccd84f1bb980f0ce73652e957f
b3bacfe6c9906d91147a14443abb6274

Thanks to @Kmc2000 for the original sprites. I had asked some weeks ago several times who made them but never got an actual answer.

🆑 Xantholne
add: New Nanomesh Combat Uniforms are now available in the loadout menu under the uniforms section for just about every job!
/🆑

### CODE REVIEW

**Comment by shazbot194** [comment]
I have to say, the top blue/light blue/cyan? one reminds me of the suits in prey a little.

**Comment by taukausanake** [comment]
I... don't know if these look really good or really bad. On the one side there is a lot of detail and blending put into each sprite but on the other I don't feel the style really fits with everything else

**Comment by Bram961** [comment]
Nanofibre actually becoming canon? yes please. some side and back shots would be nice.

**Comment by Jovaniph** [comment]
It'll be nice if these replaced the bland jumpsuits.

File: code/modules/clothing/under/jobs/civilian.dm [hunk 1]
```diff
@@ -207,9 +278,25 @@
...
 
+/obj/item/clothing/under/rank/miner/ncu
+	desc = "A Nanomesh Combat Uniform. This one is meant for a miner."
+	name = "shaft miner's nanomesh combat uniform"
+	icon_state = "ncucargo"
+	item_state = "ncucargo"
+	item_color = "ncucargo"
+	flags_size = ONESIZEFITSALL
+
+
```
Comment (KasparoVy) [inline code/modules/clothing/under/jobs/civilian.dm]: Extra newline?

File: code/modules/clothing/under/jobs/security.dm [hunk 2]
```diff
@@ -73,6 +73,16 @@
...
 
+/obj/item/clothing/under/rank/security/ncu
+	name = "security officer's nanomesh combat uniform"
+	desc = "A Nanomesh Combat Uniform. This one is fitted for Security personnel."
+	icon_state = "ncursecurity"
+	item_state = "ncursecurity"
+	item_color = "ncursecurity"
+	armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	flags_size = ONESIZEFITSALL
+
```
Comment (KasparoVy) [inline code/modules/clothing/under/jobs/security.dm]: Extra newline?

File: code/modules/clothing/under/jobs/security.dm [hunk 3]
```diff
@@ -116,6 +126,16 @@
...
+/obj/item/clothing/under/rank/head_of_security/ncu
+	name = "head of security nanomesh combat uniform"
+	desc = "A Nanomesh Combat Uniform. This one is fitted for Security personnel."
+	icon_state = "ncurhos"
+	item_state = "ncurhos"
+	item_color = "ncurhos"
+	armor = list(melee = 10, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0)
+	flags_size = ONESIZEFITSALL
+
+
```
Comment (KasparoVy) [inline code/modules/clothing/under/jobs/security.dm]: Extra newline?

**Comment by KasparoVy** [comment]
Just going to say straight up that these sprites look bloody fantastic

**Comment by Spartan6** [comment]
My only complaint is that the badges are not clear enough. This problem was corrected in jumpsuits by having then usually an entirely different colour. CMO has their jacket and the RD is just a hard to spot snowflake, but CE was white, HoS was black and Captain was blue all over.

I'd say try and make the heads stand out more.

**Comment by FalseIncarnate** [comment]
Nice sprites, but for the sake of reference, mind listing the job(s) of the suits in relation to that second picture? Would be easier for people to reference which one they are giving feedback on, instead of being like "row 3 col 3 looks like it has a red `I` on it."

At a guess I want to say the first picture is organized by department, but that red one in with the purples (guessing robotics?) does make me think it is a sec suit.

**Comment by scrubmcnoob** [comment]
Interesting sprites, cool idea, but why combat in the name? It's research station, not a battle station.
Maybe take combat out and replace it with something, maybe a word referencing to leisure or comfort or robustness in the fabric? Doesn't quite make sense that all jobs can get 'combat' uniforms.

**Comment by pinatacolada** [comment]
nanomachines son

**Comment by Xantholne** [comment]
> It'll be nice if these replaced the bland jumpsuits.
If merged and used enough along with a lot of other prereqs it's possible.

> My only complaint is that the badges are not clear enough.
Yeah I'm looking at how to make them more standy outty. This was after some 18 hours of reshading redesigning etc so I'd like to hear what others think about them after that long

> Nice sprites, but for the sake of reference, mind listing the job(s) of the suits in relation to that second picture?
I'll try to get some relevant IDs on them and take another. The only issue there is where shaftminer/cargotech enginer/mechanic and chef/bartender currently share the same ones until I can  come up with more unique ones probably in part 2.

> Interesting sprites, cool idea, but why combat in the name?
The idea was these would be uniforms used on NDVs and non-ert crews by Nanotrasen to be similar with how SolGov for instance has their own uniforms for their military bracket. It could be possible to remove Combat in the name and leave it at Nanomesh Uniform; I am not sure what people would prefer more. If it's left as NCU it also lets people have developed characters that serve with NT off space stations as well, along with adding abilities for say; The NDV Icarus sends a team over to board the NSS Cyberiad for X reason(s).

File: code/modules/clothing/under/jobs/medsci.dm [hunk 4]
```diff
@@ -92,6 +141,15 @@
...
 
+/obj/item/clothing/under/rank/virologist/ncu
+	desc = "A Nanomesh Combat Uniform. This one is fitted for Medical personnel."
+	name = "virologist's nanomesh combat uniform"
+	icon_state = "ncuvirologywhite"
+	item_state = "ncuvirologywhite"
+	item_color = "ncuvirologywhite"
+	permeability_coefficient = 0.50
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 10, rad = 0)
+	flags_size = ONESIZEFITSALL
```
Comment (Birdtalon) [inline code/modules/clothing/under/jobs/medsci.dm]: You should leave a new line for consistency.

File: code/modules/clothing/under/jobs/medsci.dm [hunk 5]
```diff
@@ -92,6 +141,15 @@
...
 /obj/item/clothing/under/rank/virologist/skirt
```
Comment (Birdtalon) [inline code/modules/clothing/under/jobs/medsci.dm]: You should have a new line here for consistency.

**Comment by FalseIncarnate** [comment]
Could change it to Nanomesh Textile Uniform (NTU) which doesn't directly imply a combat-oriented nature, clearly denotes the uniform as an "alternate" material, and also looks like it has Nanotrasen in the name.

**Comment by MarcellusPye** [comment]
I'ed add these to the relevent job closets like the other alt job uniforms for consistency.

**Comment by Xantholne** [comment]
Hm, yeah that could work.

File: code/modules/clothing/under/jobs/medsci.dm (hunk 5 above)
Comment (Xantholne) [inline code/modules/clothing/under/jobs/medsci.dm]: I have yet to look it over for fixing but I expected this much stuff when coding.

**Comment by SamHPurp** [comment]
Nanotrasen are cheap bastards. 

It just doesn't fit the lore for these fancy outfits to be used.

**Comment by taukausanake** [comment]
I agree with @SamHPurp but I also agree with the desire to different ourselves from other codebases in whatever way possible. Then again the normal jumpsuits are kind of iconic and mean something. I rather not have these replace the old suits

**Comment by uraniummeltdown** [comment]
No way should these replace the classic job jumpsuits, adding them alongside is fine though. They don't really look that great to me, it's just black with a few pixels of color in, don't see the hype

**Comment by Xantholne** [comment]
They aren't replacing the original jumpsuits it's a loadout. They'd only replace them if they where used more than.

**Comment by MarcellusPye** [comment]
Again I'ed put them in job lockers alongside the spare jumpsuits/skirts for consistency.

**Comment by Kmc2000** [comment]
@Xantholne

**Comment by Kmc2000** [comment]
For reference/proof here is the PR I made on FTL:
https://github.com/FTL13/FTL13/pull/659

You may also want to port the jackets I made to go with these:
https://github.com/FTL13/FTL13/pull/678

**Comment by Xantholne** [comment]
I was likely going to for consistency in a later PR when every role actually had it's own. Currently there's ~8 without and 3 shared with other jobs with other things I'd like to add to.

**Comment by Kmc2000** [comment]
Alright thanks for crediting me. If you want a hand making some new job specific stuff to go with them (for consistency) my discord is Kmc#2841

**Comment by Aurorablade** [comment]
why closed?

**Comment by taukausanake** [comment]
Explosions

**Pull request closed without merging**